### ISSUE TITLE
댓글을 남길 수 없습니다.

### ISSUE THREAD

**Issue by johngrib** [neoarc/neoarc.github.io#2]
댓글을 남기려 하면 다음과 같이 utterance 에서 경고문을 보여줍니다. 그래서 댓글을 남길 수 없었어요.

![image](https://user-images.githubusercontent.com/1855714/108502048-8b9fed00-72f5-11eb-8e0c-cd44fb8e3673.png)

**Comment by neoarc** [comment]
@johngrib 감사합니다. 

이 앞의 이슈로 올라온 댓글은 수동으로 남겨주신 건가보네요.
(순간 햇갈렸습니다. Issue #1은 댓글, Issue #2는 댓글 불가 이슈)

이래저래 아직 손 볼 곳들이 남아있는듯 하네요. 
누군가가 벌써 댓글 버튼을 누른줄은 상상도 못했습니다. 😃

**Comment by johngrib** [comment]
@neoarc 네. https://github.com/neoarc/neoarc.github.io/issues/1#issuecomment-782032653 이건 제가 수동으로 남긴 글입니다 :-)
